app/src/main/res/values/strings.xml

```diff
@@ -357,4 +359,8 @@
     <string name="call_for_speakers">Call For Speakers</string>
     <string name="submit_proposal">Submit Proposal</string>
 
+    <!--Facebook auth-->
+    <string name="facebook_app_id">{Facebook app id}</string>
+    <string name="fb_login_protocol_scheme">{Login protocol scheme}</string>
```

Comment:
@anhanh11001 I have removed the API key. Please add the following and test it - 
```xml
<string name="facebook_app_id">1240102032819527</string>
<string name="fb_login_protocol_scheme">fb1240102032819527</string>
```

Reply:
This is not configurable at all and will break the release process. You don't expect us to pull the code, replace the value and build the app each time right?

Reply:
Also, if the values are not present, the button should not be shown

Reply:
> This is not configurable at all and will break the release process. You don't expect us to pull the code, replace the value and build the app each time right?

Yes, I'll add these in the manifest and define them in gradle.build.